extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -226,7 +226,7 @@ export class DraggableDrawer extends AMP.BaseElement {
 
       // For displaying sticky header on mobile.
       new this.win.IntersectionObserver((e) => {
-        this.headerEl_.classList.toggle(
+        this.headerEl_?.classList.toggle(
```

Comment:
How can this happen, since `initializeListeners` is called in the layoutCallback?

Reply:
I don't think it can, It's unrelated to the bug so I think it can be removed from the PR.